vllm/v1/sample/logits_processor/interface.py

```diff
@@ -64,6 +64,15 @@ def __init__(
     ) -> None:
         raise NotImplementedError
 
+    @classmethod
+    @abstractmethod
+    def validate_params(cls, sampling_params: SamplingParams):
+        """Validate sampling params for this logits processor.
+
+        Raise ValueError for invalid ones.
+        """
+        raise NotImplementedError
```

Comment:
I think we should just `pass` here for backwards compatibility and simplicity when implementations don't care about validating args.